### PR TITLE
small update empire app

### PR DESCRIPTION
**Pull request by philbucher**
I updated the wrapper for using "doubleVectors", which is used by the IAG guys

### CODE REVIEW

File: applications/empire_application/python_scripts/empire_wrapper.py [hunk 1]
```diff
@@ -309,60 +314,94 @@ def _SetMesh(self, model_part, num_nodes, num_elements, node_coords, node_IDs, n
...
-                raise TypeError("size_of_variable could not be determined")
+            sizes_of_variables = []
+            first_node = next(iter(model_part.Nodes))
+            for variable in kratos_variables:
+                try:
+                    value = first_node.GetSolutionStepValue(variable)
+                    if (isinstance(value, float) or isinstance(value, int)): # Variable is a scalar
+                        size_of_variable = 1
+                    else:
+                        size_of_variable = len(first_node.GetSolutionStepValue(variable))
```
Comment (adityaghantasala) [inline applications/empire_application/python_scripts/empire_wrapper.py]: len(value)

**Review by adityaghantasala — COMMENTED** [review]
Everything looks good. Only thing is that I would put it in comments/description that sending something like [DISPLACEMENT, VARIABLE_X] (where variable_x is a field with only one component) is not possible. 

In this case the `sum_sizes` variable in `_get_data_field` function will be 4 and this is not valid for EMPIRE. (only vector(3) and doubleVector(6) are possible).

**Comment by philbucher** [comment]
Added the check both for sending and receiving

**Review by adityaghantasala — APPROVED** [review]
Looks good for me !

**Pull request merged**